grizli/pipeline/default_params.py

```diff
@@ -14,6 +14,12 @@
 IR_M_FILTERS = ['F098M', 'F127M', 'F139M', 'F153M']
 IR_W_FILTERS = ['F105W', 'F110W', 'F125W', 'F140W', 'F160W']
 
+NIRISS_FILTERS = ['F090W', 'F115W', 'F150W', 'F200W', 'F277W', 'F356W', 'F444W', 'F140M', 'F158M', 'F380M', 'F430M', 'F480M']
```

Comment:
Please wrap at <79 characters here and below for PEP8, perhaps wrapping at logical breaks like between F200W and F277W for NIRISS (F277W and redder are in the `FILTER` wheel while the bluer filters are in the `PUPIL` wheel....) and between the `W`, `M` and `N` filters for NIRCam.